apps/prairielearn/src/server.ts

```diff
@@ -1401,6 +1424,13 @@ export async function initExpress(): Promise<Express> {
     '/pl/course_instance/:course_instance_id(\\d+)/instance_question/:instance_question_id(\\d+)',
     (await import('./pages/studentInstanceQuestion/studentInstanceQuestion.js')).default,
   );
+  // External image capture page for an instance question. Enables users to capture and submit
+  // images from an external device (e.g. a mobile phone).
```

Comment:
Nit: IMO the comment is unnecessary, we don't typically add comments like this in `server.ts`.

```suggestion
```